plugins/vehicle/InfinitiG37/infiniti_g37.cpp

```diff
@@ -0,0 +1,184 @@
+#include "infiniti_g37.hpp"
```

Comment:
hmm so ive had some thoughts on this...

I don't think we want to add custom plugins to the dash repo... because if people want to start contributing their own plugins (say multiple people with the same vehicle) then things might get messy. And then people who don't even care about the plugin will have it built anyway

any thoughts on setting up like a repo of plugins (might be an actual good use case for submodules now haha)? Maybe we can have submodules point to individual repos, allowing them to be maintained by the actual creator?

i honestly haven't seen a repo modeled like this anyway, so not entirely sure how feasible something like that would work out

Reply:
what about templates that can be edited as per indivdual user needs?

Reply:
@rsjudka definitely agreed - This plugin has just been my test case for CAN. I do think we should include an example plugin that hooks into CAN - probably just update the test plugin?

Reply:
@jakka351 - issue is templates really can't exist, as each car manufacturer (if not each model car) is unique in how it handles CAN messages, or even what is exposed on the network.

Reply:
yeah an example would be good

im thinking we have `plugins/` in the dash repo be a submodule

in the openDsh org we'd have a plugins repo, with a directory for each category (like right now we'd have `brightness/` and `vehicle/`)

and then in each of those directories we can freely add submodules that point to individual plugins (or use this [merge subtree](https://mirrors.edge.kernel.org/pub/software/scm/git/docs/howto/using-merge-subtree.html) thing?)

this way plugins can be added more freely from dash, and users can pick and choose which individual plugins they want included